WorldsAdriftReborn/Config/ModSettings.cs

```diff
@@ -15,45 +22,75 @@ internal static class ModSettings
         public static ConfigEntry<string> localAssetPath { get; set; }
         public static ConfigEntry<string> gameServerHost { get; set; }
 
+        private static string GetCustomNamedFolder(string bepInExPluginDirectory)
+        {
+            //Because the folder under BepInEx\\plugins directory can be a custom name as per YT setup tutorial
+            var directories = Directory.GetDirectories(bepInExPluginDirectory);
+
+            if(directories !=null && directories.Count() == 0)
+            {
+                throw new Exception("BepInEx file structure not setup correctly, expected BepInEx\\plugins\\YOUR_MOD_FOLDER_HERE");
+            }
+
+            string modFolder = directories.First();
```

Comment:
This will be an issue if there is more than one mod installed. Also sorry for taking so long to react :/